Agents/CEAAgent/README.md

```diff
@@ -20,10 +20,10 @@ You will need to provide your credentials (github username/personal access token
 The agent is designed to run in the stack. To start the stack, spin up the [Stack Manager](https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager).
 
 ### Access Agent Set Up
-The agent is designed to use the [JPS Access Agent](https://github.com/cambridge-cares/TheWorldAvatar/tree/1579-dev-stackerise-cea-agent/JPS_ACCESS_AGENT). Spin the access agent up as part of the same stack as spun up by the Stack Manager.
+The agent is designed to use the JPS Access Agent (```../AccessAgent```). Spin the access agent up as part of the same stack as spun up by the Stack Manager.
```

Comment:
Sorry, what I meant was to write `[Access Agent](../AccessAgent)`. This will be rendered correctly in markdown as a hyperlink. (And please, no "JPS", anywhere, ever again...)

Reply:
Then, should I also removed JPS from the directory and the package name?

Reply:
Only if it can be done easily (without breaking everything). Otherwise, the policy is not to fix such things retrospectively.